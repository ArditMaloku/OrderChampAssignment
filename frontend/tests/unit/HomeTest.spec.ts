import { shallowMount } from '@vue/test-utils';
import HomeComponent from '@/views/Home/index.vue';

describe('HomeComponent', () => {
    it('renders without crashing', () => {
        const comp = shallowMount(HomeComponent);
        expect(comp.html()).toMatchSnapshot();
    });
});
