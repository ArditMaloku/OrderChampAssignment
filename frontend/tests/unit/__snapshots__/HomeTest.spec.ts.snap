// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent renders without crashing 1`] = `
<div class="Home">
  <hero-component-stub></hero-component-stub>
  <flowers-layout-stub></flowers-layout-stub>
</div>
`;
